addons/website_sale/models/res_config_settings.py

```diff
@@ -28,14 +28,15 @@ class ResConfigSettings(models.TransientModel):
     module_website_sale_wishlist = fields.Boolean("Wishlists")
     module_website_sale_comparison = fields.Boolean("Product Comparison Tool")
     module_website_sale_autocomplete = fields.Boolean('Address Autocomplete')
+    module_website_sale_send_abandoned_cart_email = fields.Boolean('Abandoned Email', related='website_id.send_abandoned_cart_email', readonly=False)
```

Comment:
FYI fields starting with `module_` for the `res.config.settings` model are special fields controlling the state of a module, in this case there is no module named `website_sale_send_abandoned_cart_email` so it should be named the same as in the `website's` model.

```suggestion
    send_abandoned_cart_email = fields.Boolean('Abandoned Email', related='website_id.send_abandoned_cart_email', readonly=False)
```